packages/commonwealth/Procfile

```diff
@@ -3,4 +3,4 @@ consumer: cd packages/commonwealth && node --import=extensionless/register --max
 evm-ce: cd packages/commonwealth && node --import=extensionless/register --max-old-space-size=$(../../scripts/get-max-old-space-size.sh) build/server/workers/evmChainEvents/startEvmPolling.js
 knock: cd packages/commonwealth && node --import=extensionless/register --max-old-space-size=$(../../scripts/get-max-old-space-size.sh) build/server/workers/knock/knockWorker.js
 message-relayer: cd packages/commonwealth && node --import=extensionless/register --max-old-space-size=$(../../scripts/get-max-old-space-size.sh) build/server/workers/messageRelayer/messageRelayer.js
-release: cd packages/commonwealth && node --import=extensionless/register --max-old-space-size=$(../../scripts/get-max-old-space-size.sh) build/server/scripts/releasePhaseEnvCheck.js && npx sequelize-cli db:migrate --config server/sequelize.json
+release: cd packages/commonwealth && node --import=extensionless/register --max-old-space-size=$(../../scripts/get-max-old-space-size.sh) build/server/scripts/releasePhaseEnvCheck.js && npx sequelize-cli db:migrate --config server/sequelize.json && node --import=extensionless/register build/server/scripts/purgeCloudflareCache.js
```

Comment:
Cache clearing should only come last if we are 100% sure that it will never throw. It may be safer to place it before migrations just to be safe.

Reply:
It will never throw since we are always catching any potential error and logging instead of throwing.

Maybe I am missing how the flow works, but we would only deploy the new front end after the release command runs right? Basically it should be done at the end or else we could pollute the cache with old frontend code if we are running a long migration.

Reply:
Yea, I was only mentioning the ordering here given the discussion on `process.exit(1)` and `dispose()();` above. In between the time that the cache purge request completes and the new front-end is deployed, many requests could go through and repopulate the cache with the old app files.

Reply:
Hmm, do you have any suggestions where we could put it so that it only runs after the release is complete?

Reply:
Ok we decided that we will try running the script like this, and if we still get caching issues figure out how to run the script after the deployment